src/modules/checkIsValidUserInput.js

```diff
@@ -9,9 +10,25 @@
  * @return {boolean} - True if the user input is valid, false otherwise
  */
 function checkIsValidUserInput(userInput) {
-  /* Write your code here */
+  const uniqueNumbers = new Set(userInput);
+
+  if (userInput.length !== 4) {
+    return false;
+  }
+
+  if (isNaN(+userInput)) {
+    return false;
+  }
+
+  if (+userInput < 1000) {
+    return false;
+  }
+
+  if (uniqueNumbers.size !== userInput.length) {
+    return false;
+  }
+
+  return true;
```

Comment:
U can write this in one line 

```
function checkIsValidUserInput(userInput) {
  return userInput.length === 4
      && !isNaN(+userInput)
      && +userInput >= 1000
      && new Set(userInput).size === 4;
}
```

Reply:
> U can write this in one line
> 
> ```
> function checkIsValidUserInput(userInput) {
>   return userInput.length === 4
>       && !isNaN(+userInput)
>       && +userInput >= 1000
>       && new Set(userInput).size === 4;
> }
> ```

Yes, I can do this, but it seems to me that the option with if is more readable, so it is more optimal from a calculating point of view that you do not have to check all the conditions.

Thank you for your comment 👍 . I would like to know what the mentor thinks about this

Reply:
This is purely personal and has no performance impact, so whatever (and your team) is up to.
In the professional environment you have to comply to the code styling approved together with your team.